PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -343,8 +343,8 @@ def toolCode(self, process):
 
         # recompute the MET (and thus the jets as well for correction) from scratch
         if recoMetFromPFCs:
-            self.recomputeRawMetFromPfcs(process, 
-                                         pfCandCollection, 
+            self.recomputeRawMetFromPfcs(process,
+                                         pfCandCollection,
```

Comment:
this could be change to
"puppiForMET" if pfCandCollection=="puppi" else pfCandCollection
to fix the reported issue